pkg/apis/certmanager/v1alpha1/types.go

```diff
@@ -81,6 +83,11 @@ type IssuerSpec struct {
 }
 
 type IssuerConfig struct {
+	// Certificate default Duration
+	Duration time.Duration `json:"duration,omitempty"`
+	// Certificate renew before expiration duration
+	RenewBefore time.Duration `json:"renewBefore,omitempty"`
```

Comment:
:+1: this looks good to me.

We may in future want to consider switching this to some form of 'CertificatePolicy' resource (ie certificate-class), which can be associated with a set of certificates somehow. This would help admins provide cert-specific default config easily.

For now though, this seems acceptable. We can always retain this in future as the *issuer* default (where a CertificatePolicy would override the issuers defaults)